app/components/experiment-detail.js

```diff
@@ -43,6 +44,6 @@ export default Ember.Component.extend({
             clone.save().then(() => {
                 this.get('onClone')(clone);
             });
-        }
+        },
```

Comment:
In Python, yes 100% of the time (all trailing commas)

In JS the behavior for trailing commas is not well defined across browsers, and I at least prefer not to include them.